### DISCUSSION TITLE
When returning from the background, keep the last reading position when entering the background

### DISCUSSION THREAD

**Discussion by phoenixwxy** [community/community#119935] [Mobile]
### Select Topic Area

Product Feedback

### Body

When returning from the background, keep the last reading position when entering the background

It came to the top of document when I return from the background

**Comment by stevepopovich** [comment]
Hi @phoenixwxy can you provide more info here. What screen were you on? iOS or Android? What version of the app are you using?

**Reply by phoenixwxy** [reply]
ok I forgot about this
Here is the GitHub app version and my phone version

GitHub for iOS, 版本 1.157.0 (127735376)
Device: iPhone 15 Pro (iOS 17.4.1)


By the way. I will upload a picture about this

**Reply by phoenixwxy** [reply]
I can reproduce this problem stably in this repository interface

I don't know the reason of this problem is the README is too long or because Chinese😩😩😩

![image](https://github.com/community/community/assets/32839988/182d7e41-2a3b-4f46-a59a-7d1b9687e4b4)

![image](https://github.com/community/community/assets/32839988/7a5f5927-95dd-42d6-9ca8-2eb30e8f157d)

**Reply by stevepopovich** [reply]
Ok, thank you. I can repro this and I filed a bug internally. Thanks for the report!

**Comment by phoenixwxy** [comment]
No. I don’t set any special setting在 2024年4月17日，22:54，Steven Popovich ***@***.***> 写道：﻿
Do you have any developer settings on, like Don't keep activities?
